client/state/sites/constants.js

```diff
@@ -0,0 +1,61 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { join } from 'lodash';
+
+export const SITE_REQUEST_FIELDS = join( [
```

Comment:
is there an advantage to lodash's join over the native `Array.prototype.join`?

Reply:
Not really. At first I thought that Lodash `join` has comma (`,`) as default separator char and `Array.prototype.join` doesn't, but it's not true -- it's the comma in both cases. I'll remove the Lodash import and call.